Formula/starship.rb

```diff
@@ -35,10 +35,26 @@ def install
 
     fish_output = Utils.safe_popen_read("#{bin}/starship", "completions", "fish")
     (fish_completion/"starship.fish").write fish_output
+
+    zsh_prompt = Utils.safe_popen_read("#{bin}/starship", "init", "zsh", "--print-full-init")
+    prompt_setup = "prompt_starship_setup"
+    (zsh_function/prompt_setup).write "#{prompt_setup}() {\n#{zsh_prompt}\n}\n#{prompt_setup}"
```

Comment:
Won't naming the function this way possibly shadow an existing function a user might have, also named `prompt_setup`?

Reply:
@carlocab `prompt_setup` is the variable… the function is named `prompt_starship_setup` as per the [instructions](http://zsh.sourceforge.net/Doc/Release/User-Contributions.html#Writing-Themes).

Reply:
Yes, sorry. For some reason my eyes saw quotes after `zsh_function`.